src/main/kotlin/com/workos/webhooks/models/InvitationEvent.kt

```diff
@@ -0,0 +1,20 @@
+package com.workos.webhooks.models
+
+import com.workos.usermanagement.models.InvitationEvent
+
+/**
+ * Webhook Event for `invitation.*` events.
+ */
+class InvitationEvent(
+  @JvmField
+  override val id: String,
+
+  @JvmField
+  override val event: EventType,
+
+  @JvmField
+  override val data: InvitationEvent,
```

Comment:
The naming feels a little confusing since it's the same as the event class name. Perhaps the data class should be something like `InvitationEventData`?